controllers/match.js

```diff
@@ -74,4 +74,25 @@ exports.getResultsByDates = async (req, res) => {
   } catch (error) {
     res.status(400).json({ error: error.message });
   }
+};
+
+// Sample query: http://localhost:8080/espn/getMatchesByDate?date=September%207,%202023
```

Comment:
not good practice to have date in this format in the query param.
date in the query param should be in the same format as the getGamesByDates api